pages/api/lockerbox/retrieve.ts

```diff
@@ -0,0 +1,26 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+import type { NextApiRequest, NextApiResponse } from 'next'
+import prisma from '../../../lib/prisma'
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  if (req.method === 'POST') {
+    try {
+      let { lockerId, label } = req.body
+      const lockerBox = await prisma.lockerBox.findUnique({
+        where: { specificLockerBox: { label: label, lockerId: lockerId } }
+      })
+      if (!lockerBox) {
+        res.status(400).json({ message: 'Locker not found' })
```

Comment:
If its not found we should return 404. Might also want to return the lockerId here as well so we can handle the error on the frontend nicer if we want to 
```suggestion
        res.status(404).json({ message: 'Locker not found' })
```